README.markdown

```diff
@@ -51,3 +51,14 @@ An example command would look like:
 ```
 sbt:geotrellis-spark-job> run --inputPath /tmp/cropped.tif --outputPath file:///tmp/test-catalog
 ```
+
+## EMR Configuration
+
+This project uses [sbt-lighter plugin](https://github.com/pishen/sbt-lighter) to simplify EMR cluster configuration and Spark jobs deployment. It's important to remember, that some of the Amazon instances are EBS only instances, so check out [instances types description](https://aws.amazon.com/ru/ec2/instance-types/) carefully. By default [sbt-lighter plugin](https://github.com/pishen/sbt-lighter) allocates only a 32Gb EBS volume and as a consequence you may experience issues with Spark jobs completion, so consider allocating larger EBS volumes in such cases using the following configurations that should be placed into a `build.sbt` file, or use a different instance type, that doesn't require EBS volumes usage: 
```

Comment:
This is a minor thing, but this sentence can probably be broken up so it's not as long:
``
By default, sbt-lighter plugin allocates only a 32Gb EBS volume, and as a consequence you may experience issues with Spark jobs completion. Consider allocating larger EBS volumes in such cases using the following configurations. These should be placed into a `build.sbt` file. The other option would be to use a different instance type that doesn't require EBS volumes usage.
```